### DISCUSSION TITLE
Welcome to fheroes2 Discussions!

### DISCUSSION THREAD

**Discussion by ihhub** [ihhub/fheroes2#2428] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Branikolog** [comment]
Hello. I'm the first gameplay bug-hunter of this project. Almost from the beginning of current fheroes2 team. The one, who sacrificed his eyesight, while finding bugs in game.) I will be glad to tell about our project course and  share ideas for the future.

**Comment by LeHerosInconnu** [comment]
Hello to everyone.

Heroes of Might of Magic 2 is my favorite game of all time. :1st_place_medal: 
I played Heroes 3 and its expansions a lot, and I returned to Heroes 2 after a hard disk crash, and now I only play Heroes 2.
I find the AI in Heroes 2 a lot of fun to play, and some of the design flaws of Heroes 3 are not present in Heroes 2.

Regarding fheroes2, I made a lot of proposals for the previous fheroes2 project, some of which were incorporated into the previous project.
For the new fheroes2 project, I will continue to make suggestions (paused for now) to make Heroes 2 shine even more.
I'm also reporting a bunch of bugs with 1 pixel changes and missing dots at the end of sentences. :facepalm: 

After version 1.0, I plan to make a mod for fheroes2 with small changes here and there, and add new secondary skills, new artifacts, new spells, etc.
Everything will be optional and once implemented in fheroes2, it will be possible to integrate its changes in other mods according to the players' wishes. :joy:
